BackEnd/src/business/teams.ts

```diff
@@ -44,10 +44,12 @@ export async function GetTeamsBySeasonId(seasonId: number): Promise<TeamModel[]>
 
 export async function buildRisenTeams() {
   let sheetName = 'Teams and Standings';
-
+  console.log("I AM HERE");
   let seasonsWithSheets = await GetDbActiveSeasonWithSheets();
+  console.log(seasonsWithSheets);
   for (let seasonsWithSheet of seasonsWithSheets) {
     let sheet = await GetGoogleSheet(seasonsWithSheet.googleSheetId, sheetName);
+    logger.info(sheet);
```

Comment:
Dont think we need this

Reply:
Fixedd

Reply:
Am i missing something or is this logger line still here are the "fixed" commit